src/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -11,7 +12,7 @@ export function createDirectAnswer(data: any): FeaturedSnippetDirectAnswer | Fie
     value: data.answer.value,
     relatedResult: ResultsFactory.fromDirectAnswer(data.relatedItem.data),
     verticalKey: data.relatedItem.verticalConfigId,
-    fieldType: data.answer.fieldType
+    fieldType: Object.values(BuiltInFieldType).includes(data.answer.fieldType) ? data.answer.fieldType : 'unknown'
```

Comment:
do we need to handle the case when data.answer.fieldType isn't `multi_line_text` or `rich_text` but `isFeaturedSnippetDirectAnswer` is true?

Reply:
hmm, we do expect from backend if it's a feature snippet then it should be multi_line_text or rich_text. We don't support unknown for feature snippet so I suppose we can throw an error

Reply:
updated!